packages/runtime/tsconfig.json

```diff
@@ -19,8 +19,7 @@
     "rootDir": "src",
     "declarationDir": "dist/types",
     "emitDeclarationOnly": true,
-    "jsx": "react-jsx",
-    "incremental": true
```

Comment:
🍹 Totally forgot that incremental is more useful for apps out-of-the-box rather than libraries, which require a bit more "fine-tuning". We _should_ look into this more, but we will need to think about cleaning that file whenever we need to removed along with potentially caching it in CI. We accounted for that in [BigDesign](https://github.com/bigcommerce/big-design/blob/fa36fb635610ef891a86491524aa15a8f6fe3772/packages/big-design-icons/package.json#L24) as we had a slow icon build process.